lib/task.js

```diff
@@ -309,17 +309,24 @@ function factory(
         var self = this;
 
         self.instantiateJob();
+        // Support schedulerOverrides for backwards graph definition compatibility
+        var timeout = self.definition.options.timeout;
+        if (!timeout && self.definition.options.schedulerOverrides) {
+            timeout = self.definition.options.schedulerOverrides.timeout;
+        }
```

Comment:
@yyscamper thanks, I had forgotten about the default timeout. I'll add that in.

I checked comparisons for various types, they all evaluate to false:

```
> 'test' > 0 || [] > 0 || {} > 0
false
```